### PR TITLE
Livy fix

### PR DESCRIPTION
**Pull request by stevemar**

### CODE REVIEW

**Comment by stevemar** [comment]
@rhagarty this lgtm, please review.

**Review by rhagarty — CHANGES_REQUESTED** [review]
@dilipbiswal I can't view the notebook in github which usually indicates some json issue.

**Comment by stevemar** [comment]
@rhagarty seems to render fine on nbviewer: http://nbviewer.jupyter.org/github/IBM/hdp-search-spark-recommender/blob/22236a4333d4f90a41456e2ab198f9762fdc7dd6/notebooks/solr-hdp-spark-recommender.ipynb ... maybe something is up with the github service? 🤷‍♂️

File: README.md [hunk 1]
```diff
@@ -150,33 +167,54 @@ unzip ml-latest-small.zip
 ```
 
 This code pattern is targeted to run against a multi node HDP cluster. Therefore the dataset needs to be moved to HDFS. Move the data
-to HDFS by issuing the following command:
+to HDFS by issuing the following commands:
 
 ```
-hadoop fs -put
```
Comment (rhagarty) [inline README.md:156]: @dilipbiswal - can you provide context about where this command is issued?

**Review by rhagarty — APPROVED** [review]

**Comment by rhagarty** [comment]
changes to README coming

**Pull request merged**